README.md

```diff
@@ -53,6 +53,51 @@ Application flow:
 
 When clients use up > 75% of the prior DataCap allocation, a request for additional DataCap in the form of the next tranche is automatically kicked off (via the'subsequent allocation bot'). Notaries have access to on-chain data required to verify that the client is operating in good faith, in accordance with the principles of the program, and in line with their allocation strategy outlined in the original application. 2 notaries need to approve the next tranche of DataCap to be allocated to the client. The same notary cannot sign off on immediately subsequent allocations of DataCap, i.e., you need at minimum 4 unique notaries to support your application on an ongoing basis to receive multiple tranches of DataCap. 
 
+## Application flow labels
+
+We have recently updated LDN applications with a consolidated set of labels that signify the state of the application. These updates were implemented as of April 15, 2023. To find out more about the updated look [here](https://medium.com/filecoin-plus/ldn-label-update-part-1-label-consolidation-ae2691c78702)
```

Comment:
Generally don't want timing oriented content in a readme (though I do this all the time myself, need to get better at it). I'd drop the first sentence and just restructure this bit into:
- latest version of this was released on April 15, 2023
- release notes are captured in this blog